packages/router/router.ts

```diff
@@ -147,7 +147,7 @@ export interface Router {
     key: string,
     routeId: string,
     href: string | null,
-    opts?: RouterNavigateOptions
+    opts?: RouterFetchOptions
```

Comment:
This was an underlying type bug, but unnoticed since they're effectively identical.  The changes below properly remove both `replace`/`state` from `fetch()` options